packages/amplify-graphql-conversation-transformer/src/__tests__/__snapshots__/amplify-graphql-conversation-transformer.test.ts.snap

```diff
@@ -161,18 +160,18 @@ export function request(ctx) {
     selectionSet,
   };
   const currentMessageId = ctx.stash.defaultValues.id;
+
   const modelConfiguration = {
-  modelId: 'anthropic.claude-3-haiku-20240307-v1:0',
-  systemPrompt: "You are a helpful chatbot. Answer questions to the best of your ability.",
-  inferenceConfiguration: {"temperature":0.5,"topP":0.9,"maxTokens":100},
-};
+    modelId: "anthropic.claude-3-haiku-20240307-v1:0",
+    systemPrompt: "You are a helpful chatbot. Answer questions to the best of your ability.",
+    inferenceConfiguration: {"temperature":0.5,"topP":0.9,"maxTokens":100},
+  };
 
   const clientTools = args.toolConfiguration?.tools?.map((tool) => {
     return { ...tool.toolSpec };
   });
-  const toolsConfiguration = {
-  clientTools
-};
+  const dataTools = undefined;
+  const toolsConfiguration = { dataTools, clientTools };
```

Comment:
This is expected if no tools are provided in the conversation directive. It was part of reworking the resolver function generation to stop constructed fragments of the JS resolver runtime code as strings within the transformer.